parm/config/gfs/config.base

```diff
@@ -435,4 +435,7 @@ if [[ ${DO_JEDIATMVAR} = "YES" ]]; then
    export DO_VMINMON="NO"     # GSI minimization monitoring
 fi
 
+# If starting from warm restarts at the end of the IAU window. Used in GEFS when using Replay ICs
+export END_OF_IAU_START="false"
```

Comment:
A well-rounded way would be to define an `offset` from `current_cycle`.  For replay ICs, the offset is `+3h`.  For cycling with IAU the offset is `-3h`.  If offset is `0h`, the warm starts can be expected to be valid at the `current_cycle`.